### ISSUE TITLE
Non-critical: Class never used 

### ISSUE THREAD

**Issue by ArthurAlexandersson** [Chimbobedeh/TDA553-lab1#13]
Good implementation, used togheter with DrawableVehicle adheres to the MVC pattern (keeping all view related stuff away from model) and OCP making it easy to add new DrawableVehicles. 

https://github.com/Chimbobedeh/TDA553-lab1/blob/9f43843b90b14fae1bfde454a56e2b9146328eed/DrawableVehicleFactory.java#L5

**Comment by rilind404** [comment]
We implemented it as a suggestion from one of the TA's. But writing the class-file brought alot more bugs and issues that were both VSC issues (which we troubleshooted and couldn't resolve without rewriting the whole file exactly the same) and issues with implementation.

The implementation issues resulted in alot more work needed to get this to work. With the limited time we have currently on wrapping things up, we decided to go with the "easier" route and just get things working. We spoke to TA about this, and he agreed that it doesn't look pretty, and is not optimal in the long run, but should be acceptable for this lab.

**Comment by ArthurAlexandersson** [comment]
Indeed, fixing this now is not necessary, but as long as you've understood how the program could be implemented in a better way then that's good enough.
